packages/react-storage/src/components/StorageBrowser/controls/hooks/useDataTable.ts

```diff
@@ -152,12 +152,7 @@ export const useDataTable = (): DataTableProps | null => {
       .flat();
   }, [sortState, tableData]);
```

Comment:
possibly unrelated to the PR, but how does refresh affect sort state for the table?

Reply:
I'm not sure what the right behavior should be but when hitting refresh today, the sort state is maintained